packages/yuudachi/src/functions/reports/transformReport.ts

```diff
@@ -0,0 +1,46 @@
+import type { Snowflake } from 'discord.js';
+import type { Report } from './createReport.js';
+
+export interface RawReport {
+	guild_id: Snowflake;
+	report_id: number;
+	type: number;
+	status: number;
+	message_id?: Snowflake | undefined | null;
+	channel_id?: Snowflake;
+	target_id: Snowflake;
+	target_tag: string;
+	author_id: Snowflake;
+	author_tag: string;
+	mod_id?: Snowflake | undefined | null;
+	mod_tag?: string | undefined | null;
+	reason: string;
+	attachment_url?: string | undefined | null;
+	log_message_id?: Snowflake | undefined | null;
+	ref_id?: number | undefined | null;
+	updated_at?: string | undefined | null;
+	created_at: string;
+}
```

Comment:
Something about this isn't right. The values you get back from the database can't be `undefined` or missing `?`, so it should either be there, if `not null` or `| null` but not `undefined`.

Check `RawCase`

Reply:
Fixed in 33743ae